zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/GatewayGrpcService.java

```diff
@@ -55,9 +56,12 @@
 
 public class GatewayGrpcService extends GatewayImplBase {
   private final EndpointManager endpointManager;
+  private final UserServices userServices;
 
-  public GatewayGrpcService(final EndpointManager endpointManager) {
+  public GatewayGrpcService(
+      final EndpointManager endpointManager, final UserServices userServices) {
     this.endpointManager = endpointManager;
+    this.userServices = userServices;
```

Comment:
❓ Why do we need the `UserService` instance here? From what I can see, it's only used in the `AuthenticationInterceptor` creation inside the `Gateway` class.

Reply:
Just a leftover 👍